src/cdrizzleapi.c

```diff
@@ -452,20 +474,22 @@ invert_pixmap_wrap(PyObject *self, PyObject *args)
     double *xy, *xyin;
     npy_intp *ndim, xyin_dim = 2;
 
-    xyin = (double *) malloc(2 * sizeof(double));
+    xyin = (double *)malloc(2 * sizeof(double));
 
-    if (!PyArg_ParseTuple(args,"OOO:invpixmap", &pixmap, &xyout, &bbox)) {
-      return NULL;
+    if (!PyArg_ParseTuple(args, "OOO:invpixmap", &pixmap, &xyout, &bbox)) {
+        return NULL;
     }
 
-    xyout_arr = (PyArrayObject *)PyArray_ContiguousFromAny(xyout, NPY_DOUBLE, 1, 1);
+    xyout_arr =
+        (PyArrayObject *)PyArray_ContiguousFromAny(xyout, NPY_DOUBLE, 1, 1);
     if (!xyout_arr) {
-      return PyErr_Format(gl_Error, "Invalid xyout array.");
+        return PyErr_Format(gl_Error, "Invalid xyout array.");
     }
 
-    pixmap_arr = (PyArrayObject *)PyArray_ContiguousFromAny(pixmap, NPY_DOUBLE, 3, 3);
+    pixmap_arr =
+        (PyArrayObject *)PyArray_ContiguousFromAny(pixmap, NPY_DOUBLE, 3, 3);
```

Comment:
If another line is needed for a function call, I think it's better to put the function parameters on the next line, rather than entire assignment.